{"name": "test_zh_stsz_clya_yabj_05", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_022_zh_xtsz_clya_yabj.Test_zh_stsz_clya_yabj object at 0x000002442709E390>\naccess_web = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"53dd9c390b28b1c962e76d6481f201f1\")>, <PageObjects.zh_001.dl_...bbpz object at 0x0000024427020390>, <PageObjects.zh_007_xtsz_lkpz.Test_zh_xtsz_lkpz object at 0x00000244270205F8>, ...)\n\n    def test_zh_stsz_clya_yabj_05(self, access_web):\n        time.sleep(4)\n        # 搜索框\n        access_web[20].zh_xtsz_clya_yabj_027(\"脚本添加\")\n        # 搜索框确定\n        access_web[20].zh_xtsz_clya_yabj_030()\n        # 列表最后一个\n>       access_web[20].zh_xtsz_clya_yabj_06()\n\nTestCases\\test_022_zh_xtsz_clya_yabj.py:105: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nPageObjects\\zh_022_xtsz_clya_yabj.py:89: in zh_xtsz_clya_yabj_06\n    WebDriverWait(self.driver, 30).until(EC.visibility_of_element_located(loc.zh_xtsz_clya_yabj_lbzhyg))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"53dd9c390b28b1c962e76d6481f201f1\")>, method = <selenium.webdriver.support.expected_conditions.visibility_of_element_located object at 0x000002442709E898>, message = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nd:\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "start": 1597725035496, "stop": 1597725070097, "uuid": "fd3ec270-3c29-4d76-bbaa-c3e26fdb7b68", "historyId": "6515c30d87402f3119d0a83f6fd8dcf6", "testCaseId": "b7cad47fe9ec334b3fe8281a46eb6fb9", "fullName": "TestCases.test_022_zh_xtsz_clya_yabj.Test_zh_stsz_clya_yabj#test_zh_stsz_clya_yabj_05", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('access_web')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_022_zh_xtsz_clya_yabj"}, {"name": "subSuite", "value": "Test_zh_stsz_clya_yabj"}, {"name": "host", "value": "DESKTOP-IB24K0P"}, {"name": "thread", "value": "13640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_022_zh_xtsz_clya_yabj"}]}